src/DN404.sol

```diff
@@ -821,6 +837,7 @@ abstract contract DN404 {
             _set(owned[to], n, uint32(id));
             _setOwnerAliasAndOwnedIndex(oo, id, _registerAndResolveAlias(toAddressData, to), n);
         }
+        _afterNFTTransfer(from, to, id);
         /// @solidity memory-safe-assembly
         assembly {
             // Emit the {Transfer} event.
```

Comment:
Gonna place the `_afterNFTTransfer` before the ERC20 Transfer event for consistency with the other functions.